### DISCUSSION TITLE
Remove some label prefixes

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#6234] [Discussion]
I see why it was done, but and don't think obvious labels like bug or enhancement need prefixes. It just makes them slightly harder to search for.

**Comment by WorldLanguages** [comment]
cc @Hans5958

**Comment by DNin01** [comment]
Typing `: b` returns `type: bug` and typing `enh` returns `type: enhancement`. I think the searchability is fine.

**Reply by Samq64** [reply]
I didn't know the the acual label filtering was that good. That's not really the problem though. It's partly forgetting the quotes since I tend to just type the labels when searching for more than one, but the length is also a little annoying.

**Reply by Samq64** [reply]
I guess that the case with all labels with spaces though.

**Comment by DNin01** [comment]
It does mean they're near the bottom of the list when sorted alphabetically, though.

**Comment by Hans5958** [comment]
I'd rather have consistency than simplicity. We have too much labels to have some without the prefix.
